src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
+const DEFAULT_HEALTH = 100;
+const DAMAGE_VALUE = 50;
+
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = DEFAULT_HEALTH) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  removeDead() {
```

Comment:
The `removeDead` method should be static since it operates on the static `Animal.alive` array and does not depend on instance-specific data.